### PR TITLE
Merge v4 to main

### PR DESCRIPTION
**Pull request by lahma**
I've rebased the current v4 and minor cleanup. @drieseng @jafin now is a good time to check out the commit history and yell STOOOP if you find something concerning. I can rewrite messages and commits if needed, when I get green light I can merge to main. I've branched 3.x for maintenance now.

### CODE REVIEW

File: src/Quartz.Jobs/Quartz.Jobs.csproj [hunk 1]
```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <Description>Quartz.NET Jobs; $(Description)</Description>
     <TargetFrameworks>net462;net472;netstandard2.0</TargetFrameworks>
```
Comment (jafin) [inline src/Quartz.Jobs/Quartz.Jobs.csproj:4]: Duplicate line to remove.

File: src/Quartz.Plugins.TimeZoneConverter/Quartz.Plugins.TimeZoneConverter.csproj [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>net462;net472;netstandard2.0</TargetFrameworks>
```
Comment (jafin) [inline src/Quartz.Plugins.TimeZoneConverter/Quartz.Plugins.TimeZoneConverter.csproj:3]: Duplicate TargetFramework line to remove.

**Review by jafin — CHANGES_REQUESTED** [review]
Looked over the content I'm familiar with LGTM,  a few duplicate TargetFramework attributes to remove is all I see to correct.

File: src/Quartz.Examples/Quartz.Examples.csproj [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>net6.0;net462</TargetFrameworks>
```
Comment (jafin) [inline src/Quartz.Examples/Quartz.Examples.csproj:3]: Duplicate TargetFramework line remove.

File: src/Quartz.Plugins/Quartz.Plugins.csproj [hunk 4]
```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <Description>Quartz.NET Plugins; $(Description)</Description>
     <TargetFrameworks>net462;net472;netstandard2.0</TargetFrameworks>
```
Comment (jafin) [inline src/Quartz.Plugins/Quartz.Plugins.csproj:4]: Duplciate TargetFrameworks remove.

File: src/Quartz.Serialization.Json/Quartz.Serialization.Json.csproj [hunk 5]
```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>net462;net472;netstandard2.0</TargetFrameworks>
```
Comment (jafin) [inline src/Quartz.Serialization.Json/Quartz.Serialization.Json.csproj:3]: Duplicate targetFrameworks line remove.

File: src/Quartz.Server/Quartz.Server.csproj [hunk 6]
```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>net462;net6.0</TargetFrameworks>
```
Comment (jafin) [inline src/Quartz.Server/Quartz.Server.csproj:3]: Duplicate targetFrameworks line remove.

File: src/Quartz/Quartz.csproj [hunk 7]
```diff
@@ -3,6 +3,8 @@
     <AssemblyTitle>Quartz.NET</AssemblyTitle>
     <TargetFrameworkVersion>4.6.2</TargetFrameworkVersion>
     <TargetFrameworks>net462;net472;netstandard2.0</TargetFrameworks>
```
Comment (jafin) [inline src/Quartz/Quartz.csproj:5]: Duplicate targetFrameworks lines remove.

**Comment by lahma** [comment]
@jafin thanks for the review! Good catch, git csproj merge strikes once again - I think I've fixed them in force-pushed top commit.

**Review by jafin — APPROVED** [review]

**Comment by lahma** [comment]
OK it seems that to get a first bug report from 3.4 release we need to merge this into main - that's just Mr. Murphy's law. There's automation to allow releases from 3.x branch too so I'm not that worried.

**Pull request merged**